runbot_website_display/models/runbot_repo.py

```diff
@@ -9,5 +9,5 @@ class RunbotRepo(models.Model):
 
     _order = 'sequence asc'
 
-    sequence = fields.Integer()
+    sequence = fields.Integer(default=0)
```

Comment:
aha, here we go: https://github.com/OCA/OCB/blob/8.0/openerp/models.py#L2394, https://github.com/OCA/OCB/blob/8.0/openerp/fields.py#L425, and there's no "default default" set in the `Integer` class: https://github.com/OCA/OCB/blob/8.0/openerp/fields.py#L1037

Interesting, so we should always set defaults for integer fields it seems, otherwise we get NULLs in the database on installation, which messes up sorting in this case